WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingPaymentsSetupCoordinator.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+import SwiftUI
+import struct Yosemite.Site
+
+/// Coordinates navigation of the payments setup action from store onboarding.
+final class StoreOnboardingPaymentsSetupCoordinator: Coordinator {
+    enum Task {
+        case wcPay
+        case payments
+    }
+
+    let navigationController: UINavigationController
+
+    private let task: Task
+    private let site: Site
+
+    init(task: Task, site: Site, navigationController: UINavigationController) {
+        self.task = task
+        self.site = site
+        self.navigationController = navigationController
+    }
+
+    func start() {
+        // Navigation controller for the payments setup flow.
+        let modalNavigationController = WooNavigationController()
+        showSetupView(in: modalNavigationController)
+        navigationController.present(modalNavigationController, animated: true)
+    }
+}
+
+private extension StoreOnboardingPaymentsSetupCoordinator {
+    func showSetupView(in navigationController: UINavigationController) {
+        let setupController = StoreOnboardingPaymentsSetupHostingController(task: task) { [weak self] in
+            self?.showWebview(in: navigationController)
+        } onDismiss: {
+            navigationController.dismiss(animated: true)
+        }
+        navigationController.pushViewController(setupController, animated: false)
+    }
+
+    func showWebview(in navigationController: UINavigationController) {
+        let urlString: String
+        let title: String
+        switch task {
+        case .wcPay:
+            urlString = "\(site.adminURL.removingSuffix("/"))/admin.php?page=wc-settings&tab=checkout"
+            title = Localization.wcPayWebviewTitle
+        case .payments:
+            urlString = "\(site.adminURL.removingSuffix("/"))/admin.php?page=wc-admin&task=payments"
+            title = Localization.paymentsWebviewTitle
```

Comment:
Nit: Consider moving these to a `Constants` enum.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/9188/commits/c242fd06496fb15ee0f631d338b54f0320f335b3